pulp_rpm/tests/functional/constants.py

```diff
@@ -32,13 +32,17 @@
 RPM_UNSIGNED_FIXTURE_URL = urljoin(PULP_FIXTURES_BASE_URL, 'rpm-unsigned/')
 """The URL to a repository with unsigned RPM packages."""
 
-RPM_FIXTURE_COUNT = 39  # 35 Packages + 4 UpdateRecord units
-"""The total number of content units present in the standard repositories, i.e.
-:data:`RPM_SIGNED_FIXTURE_URL` and :data:`RPM_UNSIGNED_FIXTURE_URL`.
+RPM_PACKAGES_COUNT = 35
+"""The number of packages available at
+:data:`RPM_SIGNED_FIXTURE_URL` and :data:`RPM_UNSIGNED_FIXTURE_URL`
 """
 
+RPM_UPDATED_RECORD_COUNT = 4
```

Comment:
It should actually be read as "updaterecord", not "updated record".

Reply:
http://rpm-software-management.github.io/createrepo_c/python/lib.html#createrepo_c.UpdateRecord